app/views/layouts/application.html.erb

```diff
@@ -67,18 +66,33 @@
                 <li class="nav-dropdown-item"><a href="/forms/new">Forms</a></li>
               </ul>
             </li>                
-            <li>
-              <a href="#contact" class="cdc-navbar-item"><i class="fa fa-bell item-navbar-icon" aria-hidden="true"></i>
+            <li class="dropdown">
+              <a href="#notifications" class="dropdown-toggle cdc-navbar-item" id="notifications-dropdown" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><i class="fa fa-bell item-navbar-icon" aria-hidden="true"></i>
               Alerts
-              <span class="alerts-badge">1</span></a>
-            </ul>
+              <% if current_user.notifications.exists? and current_user.notifications.where('read' => false).count > 0 %>
+                <span class="caret"></span>
+                <span class="alerts-badge"><%= current_user.notifications.where('read' => false).count %></span></a>
+                <ul id="notification-menu" class="cdc-nav-dropdown">
+                </ul>
+              <% else %>
+                <span class="caret"></span></a>
+                <ul class="cdc-nav-dropdown">
+                  <ul><li class="notification-menu-item">No new notifications</li></ul>
+                </ul>
+              <% end %>
+            </li>
           </ul>
           <% end %>
         </div>
+        <% if user_signed_in? and current_user.notifications.where('read' => false).exists? %>
```

Comment:
please do not use the 'and'  keyword, && is preferable and they are not equivalent